app/components/item/Block.vue

```diff
@@ -44,7 +44,7 @@ export default {
     this.loadNewsImages()
   },
   methods: {
-    async loadNewsImages() {
+    loadNewsImages () {
```

Comment:
@baxgas why this change?

Reply:
doesn't need `async`, since it doesn't return a Promise to be resolved and doesn't have `await` for the variables init in the body, so it is not used as an async function. 
don't know why it was put in the first place like this